Code/Framework/AzGameFramework/AzGameFramework/Application/GameApplication.cpp

```diff
@@ -115,6 +116,7 @@ namespace AzGameFramework
         AZ::ComponentTypeList components = Application::GetRequiredSystemComponents();
 
 #if !defined(_RELEASE)
+        components.emplace_back(azrtti_typeid<AzNetworking::NetworkingSystemComponent>());
```

Comment:
This should be removed.
The AzNetworkingModule already sets the NetworkingSystemComponent as requried.